Userland/Applications/KeyboardMapper/KeyboardMapperWidget.cpp

```diff
@@ -126,18 +126,19 @@ void KeyboardMapperWidget::create_frame()
 void KeyboardMapperWidget::load_from_file(String filename)
 {
     auto result = Keyboard::CharacterMapFile::load_from_file(filename);
-    VERIFY(result.has_value());
-
-    m_filename = filename;
-    m_character_map = result.value();
-    set_current_map("map");
+    if (result.has_value()) {
```

Comment:
Let's use an early return style here to avoid having to indent everything:

```suggestion
    if (!result.has_value()) {
        dbgln("Failed to load character map from file {}", filename);
        return;
    }
    ...
```